lib/command.js

```diff
@@ -1126,6 +1184,59 @@ Expecting one of '${allowedValues.join("', '")}'`);
     }
   }
 
+  /**
+   * @param {Argument|Option} target
+   * @param {*} value
+   * @param {Function} handleError
+   * @api private
+   */
+
+  _handleChainError(target, value, handleError) {
+    if (target.chained && ['then', 'catch'].every(
```

Comment:
Why test for `catch` too, instead of just "thenable"?

Reply:
Because `catch` is called right after, and one could theoretically supply a thenable without a `catch` function.. The call could be replaced by `value.then(null, handleError)` or `value.then(x => x, handleError)` though.

Reply:
Oh right, thanks.